internal/pkg/cosign/tsa/signer.go

```diff
@@ -84,8 +86,14 @@ func (rs *signerWrapper) Sign(ctx context.Context, payload io.Reader) (oci.Signa
 		return nil, nil, err
 	}
 
-	// Here we get the response from the timestamped authority server
-	responseBytes, err := GetTimestampedSignature([]byte(b64Sig), rs.tsaClient)
+	// create timestamp over raw bytes of signature
+	rawSig, err := base64.StdEncoding.DecodeString(b64Sig)
```

Comment:
Feels a little gross to ask for the base64 signature then immediately decode it. I really want to push *all* encoding to the application boundaries as much as possible.

Can we add a TODO to add a `Signature()` method to `oci.Signature`?